rules/config

```diff
@@ -137,7 +137,7 @@ INCLUDE_SYSTEM_TELEMETRY = n
 INCLUDE_ICCPD = n
 
 # INCLUDE_SFLOW - build docker-sflow for sFlow support
-INCLUDE_SFLOW = y
+INCLUDE_SFLOW = n
 
 # INCLUDE_MGMT_FRAMEWORK - build docker-sonic-mgmt-framework for CLI and REST server support
 INCLUDE_MGMT_FRAMEWORK = y
```

Comment:
As per my understanding, management framework is not matured enough for wide adoption. The use case in switching world is limited right now. Are we planning to bring in unified manageability for optical devices via this daemon ?